### PR TITLE
DEV-010-Rest-Client: Adding VS code REST client requests

### PR DESCRIPTION
**Pull request by Prashanga**
## Rationale

Adding VS Code REST client commands under REST_requests folder. This allows to test API endpoints without having to leave VS Code.

## Advice for Reviewers & Testing Notes

- Start by adding VS REST client from the following link: 
https://marketplace.visualstudio.com/items?itemName=humao.rest-client
- To send request, just click the 'Send Request' option as shown in the screenshot

## Task Name

- DEV-010-Rest-Client

## Linting Checklist
The following is a list of linting rules that ESLint is not currently linting for. Please make sure your code conforms to this list. Examples/definitions of the rules can be found [here](https://hireup.atlassian.net/wiki/pages/viewpageattachments.action?pageId=618037301&metadataLink=true&preview=/618037301/618168365/PR_Checklist.pdf)
- [ ] No commented code
- [ ] Code Formatted nicely (Prettier)
- [ ] PR your own code before you assign reviewers

### CODE REVIEW

File: server/REST_requests/mentors/delete_by_ID_mentor.REST [hunk 1]
```diff
@@ -0,0 +1 @@
+DELETE http://localhost:3000/api/v1/mentors/5e897afeddbd502ed8d7c62a
```
Comment (promie) [inline server/REST_requests/mentors/delete_by_ID_mentor.REST:1]: I thought we're running our localhost at PORT 5000. How did you get it work on PORT 3000?

**Comment by promie** [comment]
I'm getting this error when I press "Send Request" for the GET request to get all mentors:

image

File: server/REST_requests/mentors/delete_by_ID_mentor.REST (hunk 1 above)
Comment (Prashanga) [reply server/REST_requests/mentors/delete_by_ID_mentor.REST:1]: Idk  why but my env file has port 3000

Comment (Prashanga) [reply server/REST_requests/mentors/delete_by_ID_mentor.REST:1]: The error is because the port is incorrect. Let me fix it

**Pull request merged**